copy/digesting_reader.go

```diff
@@ -23,7 +23,7 @@ type digestingReader struct {
 func newDigestingReader(source io.Reader, expectedDigest digest.Digest) (*digestingReader, error) {
 	var digester digest.Digester
 	if err := expectedDigest.Validate(); err != nil {
-		return nil, fmt.Errorf("Invalid digest specification %s", expectedDigest)
+		return nil, fmt.Errorf("Invalid digest specification %s: %w", expectedDigest, err)
```

Comment:
If we are touching this,
```suggestion
		return nil, fmt.Errorf("invalid digest %q: %w", expectedDigest, err)
```
please

Reply:
Changed as suggested.